Python/other/staircase_climbs.py

```diff
@@ -0,0 +1,34 @@
+# You are given a positive integer N which represents the number of steps in a staircase. 
+# You can either climb 1 or 2 steps at a time. Write a function that returns the number of unique ways to climb the stairs.
+'''
+Input: n = 4
+Output: 5
+(1, 1, 1, 1), (1, 1, 2), (2, 1, 1), (1, 2, 1), (2, 2)
+
+Input: n = 5
+Output: 8
+(1, 1, 1, 1, 1), (2, 1, 1, 1), (1, 2, 1, 1), (1, 1, 2, 1), (1, 1, 1, 2), (2, 2, 1), (2, 1, 2), (1, 2, 2)
+'''
+# Time Complexity - O(n^2)
+# Space Complexion - O(1)
+def staircase(n: int) -> int:
+  if n <= 3:
+    return n
+  else:
+    return staircase(n-1)+staircase(n-2)
+
```

Comment:
@hemraj4545 remove unnecessary lines

Reply:
removed else statement and idented the last return statement.